src/server/pkg/lokiutil/lokiutil.go

```diff
@@ -70,16 +65,10 @@ func QueryRange(ctx context.Context, c *loki.Client, queryStr string, from, thro
 			return nil
 		}
 		from = from.Add(time.Nanosecond)
-		// check if the context has been cancelled
```

Comment:
I think this is no longer required with being able to pass the context through to the request?